pandas/io/html.py

```diff
@@ -1080,6 +1081,10 @@ def read_html(
             "data (you passed a negative value)"
         )
     validate_header_arg(header)
+
+    if isinstance(io, Path):
+        io = stringify_path(io)
```

Comment:
you can remove the if-condition, `stringify_path`does that internally. Please mention "closes #37705" in your PR so that github will close this issue when this PR gets merged.

Reply:
Got it! Thanks for the comment!!